### ISSUE TITLE
Feature request - AVIF output

### ISSUE THREAD

**Issue by danmons** [brion/hdrfix#8]
Thank you for this tool! It's been a frustrating ride trying to find things that work specifically with Windows-produced JXR files (it seem like Xbox HDR screenshots also in JXR format don't use the same 32bit scRGB encoding internally).  But your tool has made that both easy and flexible in choice of tonemapping algorithms and adjustments. 

I know it's a rather big ask, but AVIF is a format that is becoming more and more useful (and compatible with far more software).  I was wondering if you would consider adding functionality to hdrfix to be able to convert Windows JXR files to AVIF format, and optionally convert them across to BT.2020+PQ too?  I know that's very different to what it does now, but that might be a feature useful to others wanting to use these frustrating JXR files in other HDR-aware applications or displays.

Thanks again for this project!

**Comment by brion** [comment]
Yeah AVIF in & out would be useful... I might not be able to get to this immediately but it'll definitely come in handy for my future plans for this tool, converting files back and forth for use in automated tooling!

**Comment by brion** [comment]
Just a note I'll want to be able to save both HDR AVIF (with only format conversion) and SDR AVIF (with the tone-mapping) so I need to add a switch rather than detecting based on format. :D

**Comment by dgrdsv** [comment]
FYI: Steam has just added lossless AVIF HDR screenshots in Windows beta client. Would be great to be able to tonemap into them JPG/PNG with HDRFix.
